firmware/controllers/system/efi_gpio.cpp

```diff
@@ -358,15 +358,11 @@ OutputPin::OutputPin() {
 }
 
 bool OutputPin::isInitialized() {
-#if EFI_GPIO_HARDWARE && EFI_PROD_CODE
-#if (BOARD_EXT_GPIOCHIPS > 0)
-	if (ext)
-		return true;
-#endif /* (BOARD_EXT_GPIOCHIPS > 0) */
-	return port != NULL;
-#else /* EFI_GPIO_HARDWARE */
+#if EFI_PROD_CODE
+	return brainPin != GPIO_UNASSIGNED;
```

Comment:
why does this work for TLE chips? let's add a comment?

Reply:
it works because brainPin is always configured, whether it's an external chip or not

Reply:
oh, brain pin not port

I am paranoid about change. this would require manual testing on MRE and that's not something I am excited about :(

Reply:
the same test is already performed when actually setting the pin in `setValue`:

![image](https://user-images.githubusercontent.com/568254/102887552-063eb000-440c-11eb-9f18-3338230fa0b5.png)

Reply:
And `OutputPin::initPin` unconditionally sets `brainPin` (no matter if unit test, on chip, external chip, etc), and `OutputPin::deInit` unconditionally sets it to `GPIO_UNASSIGNED`.